praca/praca.tex

```diff
@@ -245,48 +245,259 @@ \subsubsection{System wieloagentowy}
     \begin{figure}[h]
         \caption{Schemat działania MATSim}
-        \includegraphics[width=\textwidth]{images/matsim-workflow.png}
+        \includegraphics[width=\textwidth]{images/mopsim/matsim-workflow.png}
     \end{figure}
 \newpage
 \subsubsection{Dlaczego MATSim?}
-Przeprowadzone próby pokazały, że MATSim może być dobrym narzędziem do symulowania ruchu na drogach w Polsce, w celu zbadania zajętości MOP-ów. Co istotne, wstępne symulacje pokazały, że już jedna iteracja działania głównej pętli programu jest wystarczająca -- wynik jest bardzo zbliżony do optymalnego. Jest to zgodne z~intuicją -- w ruchu krajowym należy oczekiwać, że najkrótsza droga spośród dróg danego typu jest najbardziej optymalną, a~tworzące się korki nie są tak częste i~regularne, jak w intensywnym ruchu miejskim. Czynniki, które nas przekonały do wyboru frameworku MATSim to między innymi:
+Przeprowadzone próby pokazały, że MATSim może być dobrym narzędziem do symulowania ruchu na drogach w Polsce, w celu zbadania zajętości MOP-ów. Co istotne, wstępne symulacje pokazały, że już jedna iteracja działania głównej pętli programu jest wystarczająca -- wynik jest bardzo zbliżony do optymalnego. Jest to zgodne z~intuicją -- w ruchu krajowym należy oczekiwać, że najkrótsza droga spośród dróg danego typu jest najbardziej optymalną, a~tworzące się korki nie są tak częste i regularne, jak w intensywnym ruchu miejskim. Czynniki, które nas przekonały do wyboru frameworku MATSim to między innymi:
 \begin{itemize}
 \item licencja open-source
 \item możliwość łatwego rozszerzenia o dodatkowe funkcjonalności, wiele gotowych przykładów
 \item obszerna dokumentacja i dostęp do kodu źródłowego
 \item możliwość zrównoleglania
 \end{itemize}
 
-\subsubsection{Mopsim -- schemat działania programu}
+\subsubsection{Mopsim -- technologie}
+W programie Mopsim wykorzystano następujące technologie:
+\begin{itemize}
+\item MATSim -- opisany wyżej
+\item Java -- choć MATSim umożliwia też pisanie rozszerzeń w innym języku, zdecydowaliśmy się na wykorzystanie Javy jako języka programowania z uwagi na jej największą integrację z MATSimem. Dodatkowym atutem było nasze doświadczenie z pracy w tym języku, także w aspekcie programowania współbieżnego.
+\item OpenStreetMap -- Mopsim korzysta z map OpenStreetMap do opisu sieci drogowej. Jest to projekt społeczności internetowej mający na celu stworzenie darmowej, swobodnie dostępnej mapy całej kuli ziemskiej. Jej atutem są gotowe rozwiązania umożliwiające edycję oraz integracja z MATSimem, który posiada moduł umożliwiający konwersję na format używany do opisu dróg w symulacji.
```

Comment:
*która

Reply:
co? xd

Reply:
Aa, bo to matsim posiada moduł. No to niby ok, ale jakieś skomplikowane jest to zdanie

Reply:
ale to już było wcześniej i jest w masterze xD

Reply:
może wcześniej tego nie zauważyłam ;P 
a może "intergrację"? wtedy brzmi lepiej
a ta integracja jest też gotowym rozwiązaniem

Reply:
dobra, "integrację"